src/tests/integration/omneo/profile/list-profile-dates.test.ts

```diff
@@ -0,0 +1,62 @@
+import { describe, expect, test, afterAll } from 'vitest'
+import { Omneo } from '../../../../omneo'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+
+import { ProfileDatesAttribute } from '../../../../types'
+import randomString from '../../../lib/string/random'
+
+const omneo = new Omneo({
+  tenant: process.env.OMNEO_TENANT as string,
+  token: process.env.OMNEO_TOKEN as string
+})
+const CREATED_DATES_HANDLES : string[] = []
+const getHandle = () => { return `sdk_unit_test_dates_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('Profile dates get', () => {
+  test('SDK can get profile dates', async () => {
+    const payload: ProfileDatesAttribute = {
+      name: 'Wedding',
+      date: '2024-12-02',
+      handle: getHandle(),
+      is_recurring: true,
+      recurring_schedule: 'Yearly',
+      note: 'test notes for omneo',
+      relationship: 'Me',
+      description: 'test description for omneo'
+    }
+    await simpleOmneoRequest('PUT', `/profiles/${testProfileID}`, {
```

Comment:
Use this response to populate CREATED_DATES_HANDLES
since it's isolated from the SDK we're testing

Reply:
changed.